source/documentors/decisions/0002-choosing-rst.rst

```diff
@@ -0,0 +1,200 @@
+it using ReadTheDocs.org
+
+Nuance
+======
+
+This does not mean that all documentation needs to live in repositories nor
+that all of it must be RST.  There are many good reasons to write documentation
+on the wiki, in GitHub Issues, or as discussion forum posts.  Especially early
+on when there might be a lot of iteration.
+
+Concerns Considered
+===================
+
+* Git and GitHub are a barrier to entry for non-technical writers.
+   * We acknowledge that we will have to do training and write good
+     documentation to mitigate this for less technical users in our community.
+
+     Investments by GitHub as well as effort we'll be making in terms of
+     quickstarts and how-tos will help
+
+
+* RST is a barrier to entry.
+
+   * While RST can become very complicated, the basics of it are not much
+     different from Markdown and other simple markup languages.  A small
+     opinionated :doc:`quickreferenc <../references/quick_reference_rst>` should
+     help mitigate the major issues.
+
+* We want to be able to translate the documentation in the future.
+
+   * The story around translations is not robust but it seems to be more mature
+     than the alternatives we looked at.
+
+* Where possible we want the docs to be close to what they're documenting.
+
+   * Developer documentation is in RST already, choosing a different technology
+     requires us to revisit that decision as well which may be more costly.
+
+* We want the ability to interlink documentation and re-use common pieces of
+  documentation.
+
+   * RST a lot more built-in power and allows for easier componentization of
+     documents so that they can be re-used.  This is especially useful for user
+     facing How-Tos.  For example, we often need th do the same initial steps to
+     setup a course.  Making this re-usable allows us reduce boilerplate in our
+     documentation and ease updates.
+
+* We want to be able to publish multiple versions of documentation so that
+  moving forward it will be easier to leave published version of docs for old
+  releases.
+
+   * ReadTheDocs has support for being able to publish multiple versions of
+     documentation for a project.
+Rejected Alternatives
+*********************
+
+Markdown
+========
+
+The basics of markdown are extremely simple and easy to learn.  Because Markdown
+was built for the web, it's often supported in web based editors and many people
+are already quite familiar with it from other uses.
+
+However, there are currently `over 30 implementations`_ of the original Markdown
+reference specification.  The basic specification does not meet the needs of our
+project as it lacks many of the capabilities that we would want to build a large
+documentation project.
+
+   * It doesn't allow for relative document links. Which makes it difficult to
+     have multiple builds(local development vs production) of the same docs and
+     have the links point to local files locally and remote files when deployed.
```

Comment:
I haven't seen this restriction in any markdown I've used; relative links are just links and they work just like I expect them to.